plugins/org.eclipse.fordiac.ide.export.forte_ng/src/org/eclipse/fordiac/ide/export/forte_ng/base/BaseFBImplTemplate.xtend

```diff
@@ -59,22 +61,19 @@ abstract class BaseFBImplTemplate<T extends BaseFBType> extends ForteFBTemplate<
 			«type.internalConstVars.generateVariableDefinitions(true)»
 			
 		«ENDIF»
-		«IF !type.internalFbs.isEmpty»
-			«type.generateInternalFbDeclarations»
-			
-		«ENDIF»
+		
```

Comment:
Please avoid adding an additional empty line here. The previous statement already includes a (conditional) empty line. Removing this line should also fix the tests.

Reply:
Done

Reply:
Please leave the (conditional) empty line inside the IF block and remove the (unconditional) one outside.